appveyor.yml

```diff
@@ -1,20 +1,33 @@
 environment:
   SAXON_CP: '%TEMP%\xspec\saxon\saxon9he.jar'
   matrix:
-    # latest Saxon 9.7 version
+    # latest Saxon 9.7 version and XML Calabash
     - SAXON_VERSION: 9.7.0-15
+      XMLCALABASH_VERSION: 1.1.15-97

```

Comment:
Good point, I forgot about the bundled Saxon in XProc. I don't think it is our responsibility to test XProc with other versions of Saxon, we rely on XProc to work properly in order to use it with XSpec. Therefore I'm fine with using the bundled Saxon processor in XProc. 

My main concern was about the different behaviour between Travis and AppVeyor build results  as I wrote above (1 failed build for AppVeyor, 3 failed builds for Travis). I guess this is happening because XMLCalabash is in the matrix configuration for AppVeyor whereas it is in the ``before_script``` section in Travis. However, I can see that the test for XProc executed successfully in this [AppVeyor build](https://ci.appveyor.com/project/AirQuick/xspec/build/85-win_xproc/job/qswc898omvek4aj0) (that was the same code as [this failed build](https://ci.appveyor.com/project/AirQuick/xspec/build/85-win_xproc/job/u45y0th7mi890bdj) but with a different version of Saxon).   

My question is: are AppVeyor and Travis behaving slightly differently in the test for XProc? Feel free to highlight things I may not have noticed - as usual ;-)